File: src/main/java/fonter/handler/GuiHandler.java
package fonter.handler;

import fonter.client.gui.CustomGuiChat;
import net.minecraft.entity.player.EntityPlayer;
import net.minecraft.world.World;
import net.minecraftforge.fml.common.network.IGuiHandler;

/**
 * Created by EZtouch on 21/08/15.
 */
public class GuiHandler implements IGuiHandler
{

    public void registerKeyBindings()
    {
    }

    @Override
    public Object getServerGuiElement(int ID, EntityPlayer player, World world, int x, int y, int z)
    {
        return null;
    }

    @Override
    public Object getClientGuiElement(int ID, EntityPlayer player, World world, int x, int y, int z)
    {
        if (ID == CustomGuiChat.GUI_ID)
            return new CustomGuiChat();

        return null;
    }
}

File: src/main/java/com/eztouch/typingtools/handler/GuiHandler.kt
package com.eztouch.typingtools.handler

import com.eztouch.typingtools.client.gui.CustomGuiChat
import net.minecraft.entity.player.EntityPlayer
import net.minecraft.world.World
import net.minecraftforge.fml.common.network.IGuiHandler

class GuiHandler : IGuiHandler {

    fun registerKeyBindings() {}

    override fun getServerGuiElement(ID: Int, player: EntityPlayer, world: World, x: Int, y: Int, z: Int): Any? {
        return null
    }

    override fun getClientGuiElement(ID: Int, player: EntityPlayer, world: World, x: Int, y: Int, z: Int): Any? {
        if (ID == CustomGuiChat.GUI_ID)
            return CustomGuiChat()

        return null
    }
}